test/runtests.jl

```diff
@@ -4,6 +4,8 @@ testfiles = [ "dsp.jl", "util.jl", "windows.jl", "filter_conversion.jl",
     "filter_design.jl", "filter_response.jl", "filt.jl", "filt_stream.jl",
     "periodograms.jl", "resample.jl", "lpc.jl", "estimation.jl", "unwrap.jl"]
 
+srand(1776)
```

Comment:
Should add `Compat.Random` to the `using` list above.

Reply:
Fixed